docs/docs/atom-archive/getting-started/sections/atom-basics.md

```diff
@@ -0,0 +1,203 @@
+
+::: tip Note:
+
+Throughout the book, we will use shortcut keybindings like <kbd class="platform-mac">Cmd+Shift+P</kbd><kbd class="platform-windows platform-linux">Ctrl+Shift+P</kbd> to demonstrate how to run a command. These are the default keybindings for the platform that we detected you running.
+
+If you want to see a different platform than the one we detected, you may choose a different one by using the platform selector near the top of the page:
+
+![Platform Selector](@images/atom/platform-selector.png "Platform Selector")
+
+If the Platform Selector is not present, then the current page doesn't have any platform-specific content.
+
+If you have customized your Atom keymap, you can always see the keybinding you have mapped in the Command Palette or the Keybindings tab in the [Settings View](#settings-and-preferences).
+
+:::
+
+This search-driven menu can do just about any major task that is possible in Atom. Instead of clicking around all the application menus to look for something, you can press <kbd class="platform-mac">Cmd+Shift+P</kbd><kbd class="platform-windows platform-linux">Ctrl+Shift+P</kbd> and search for the command.
+
+![Command Palette](@images/atom/command-palette.png "Command Palette")
+
+Not only can you see and quickly search through thousands of possible commands, but you can also see if there is a keybinding associated with it. This is great because it means you can guess your way to doing interesting things while also learning the shortcut key strokes for doing it.
+
+For the rest of the book, we will try to be clear as to the text you can search for in the Command Palette in addition to the keybinding for different commands.
+
+#### Settings and Preferences
+
+Atom has a number of settings and preferences you can modify in the Settings View.
+
+![Settings View](@images/atom/settings.png "Settings View")
+
+This includes things like changing the theme, specifying how to handle wrapping, font settings, tab size, scroll speed and much more. You can also use this screen to install new packages and themes, which we'll cover in [Atom Packages](/sections/atom-packages).
+
+To open the Settings View, you can:
+
+* Use the <span class="platform-mac">*Atom > Preferences*</span><span class="platform-windows">*File > Settings*</span><span class="platform-linux">*Edit > Preferences*</span> menu item in the menu bar
+* Search for `settings-view:open` in the [Command Palette](#command-palette)
+* Use the <kbd class="platform-mac">Cmd+,</kbd><kbd class="platform-windows platform-linux">Ctrl+,</kbd> keybinding
+
+##### Changing the Theme
+
+The Settings View also lets you change the themes for Atom. Atom ships with 4 different UI themes, dark and light variants of the Atom and One theme, as well as 8 different syntax themes. You can modify the active theme by clicking on the Themes tab in the sidebar of the Settings View, or you can install new themes by clicking the Install tab.
```

Comment:
Are there some rules in English language regarding the usage numbers in text, e.g. use words for certain numbers?

Reply:
That one definitely should be fixed once moved into the new stuff. I'm not changing wording in there with the archived text.